web/src/components/Navbar/index.tsx

```diff
@@ -9,7 +9,7 @@ const StyledContainer = styled(Container)`
   justify-content: space-between;
   align-items: flex-start;
   align-content: center;
-  margin-top: 40px;
+  padding-top: 40px;
```

Comment:
`Margin` was not counting in the page height, causing it to overflow prematurely. Changing it to `padding` delayed when it would switch to scroll/overflow.